### DISCUSSION TITLE
alignment2

### DISCUSSION THREAD

**Discussion by giscus** [jamesstephenbrown/NZSWebsite#22] [Announcements]
# alignment2

Exploring the alignment problem between the individual and the collective.

https://nonzerosum.games/alignment2.html

**Comment by andrewtaneglen** [comment]
This polarisation feels like a misunderstanding of ends. You want unfettered capitalism and individualism to allow the most creative industrialists, scientists, and artists the freedom to maximise a nation's wealth and achievement? Well,  it seems like you want a system that produces the greatest number of creative industrialists, scientists, and artists. Is this a system of stark intergenerational inequality and low social mobility? No it is not, unless you believe in social determinism (e.g. poor people are eternally destined to be poor), and/or it's close relative: hardcore racism.
A proponent of capitalism and individualism who has transcended race/class-determinisms, should become convinced that we should arrange society so as to maximise each individual's potential to become a creative industrialist, scientist, and/or artist. This would entail a great deal of collectivist effort and expenditure, to ultimately achieve the greatest individualist end - they are the same end.
Even with long term sustainability issues - should we arrange society to fix these problems, or should we primarily arrange society to maximise our capacity to understand and solve problems - i.e. maximise every individual's capacity to understand and solve problems? Misalignment is also greatly reduced in the latter case.

**Reply by jamesstephenbrown** [reply]
Yeah, you want meritocracy? Let's stop pretending we already have one and make one.
